.circleci/config.yml

```diff
@@ -49,7 +46,6 @@ jobs:
             - dep-cache-{{ checksum "Gopkg.lock" }}
       - run: make docker
       - run: bin/testEnvRootMinikube.sh wait
-      - run: docker images
```

Comment:
It is useful, as it shows both the built images and all the dependencies. Not critical, but ~1 sec.